js/index.js

```diff
@@ -40,3 +42,124 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//navigation
+const navigation = document.querySelector(".container > header > nav")
+const navItems = document.querySelectorAll("header > nav > a");
```

Comment:
navItems is a NodeList created by querySelectorAll, so you can iterate through it via loops:

```
navItems.forEach((link, index) => {
     link.textContent = siteContent['nav'][`nav-item-${index+1}`];
     link.style.color = 'green';
});
```

OR

```
for (let i=0; i < links.length; i++) {
  navItems[i].textContent = siteContent['nav'][`nav-item-${i+1}`];
  navItems[i].style.color = 'green';
};
```